File: app/src/main/java/com/example/tribecrypto/crypto_watch_list/CryptoWatchlistComponent.java
package com.example.tribecrypto.crypto_watch_list;


import com.example.tribecrypto.di.component.ApplicationComponent;

import dagger.Component;

@Component(dependencies = ApplicationComponent.class)
public interface CryptoWatchlistComponent {
    void inject(CryptoWatchlistFragment fragment);
}

File: app/src/main/java/com/example/tribecrypto/crypto_watch_list/CryptoWatchlistComponent.kt
package com.example.tribecrypto.crypto_watch_list

import com.example.tribecrypto.di.component.ApplicationComponent
import dagger.Component

@Component(dependencies = [ApplicationComponent::class])
interface CryptoWatchlistComponent {
    fun inject(cryptoWatchlistFragment: CryptoWatchlistFragment)
}